smartapps/ethayer/lock-user.src/lock-user.groovy

```diff
@@ -571,6 +573,14 @@ def isActiveKeypad() {
   }
 }
 
+def isUserEnabled() {
```

Comment:
This is the same as just putting userEnabled in the original isActive logic

I'd prefer something like this...

```
  if (userEnabled == null) {
    return true
  } else if (userEnabled) {
    return false
  } else {
    return true
  }
```
this will protect people who haven't initialized this setting.

we would also need isUserEnabled() called on isActiveKeypad() to disable users for keypads.